pkg/cvo/status.go

```diff
@@ -31,6 +31,10 @@ const (
 	// and indicates the cluster is not healthy.
 	ClusterStatusFailing = configv1.ClusterStatusConditionType("Failing")
 
+	// ConditionalUpdateConditionTypeRecommended is a type of the condition present on a conditional update
+	// that indicates whether the conditional update is recommended or not
+	ConditionalUpdateConditionTypeRecommended = "Recommended"
```

Comment:
There are a few other places where you could pivot to this:

```console
$ git --no-pager grep '"Recommended"' origin/pr/961
origin/pr/961:pkg/cvo/availableupdates.go:                      if recommended := meta.FindStatusCondition(conditionalUpdate.Conditions, "Recommended"); recommended == nil {
origin/pr/961:pkg/cvo/availableupdates.go:              if recommended := meta.FindStatusCondition(conditionalUpdate.Conditions, "Recommended"); recommended == nil {
origin/pr/961:pkg/cvo/availableupdates.go:                              Type:   "Recommended",
origin/pr/961:pkg/cvo/availableupdates_test.go:                                 Type:    "Recommended",
origin/pr/961:pkg/cvo/status.go:        ConditionalUpdateConditionTypeRecommended = "Recommended"
origin/pr/961:pkg/payload/precondition/clusterversion/recommendedupdate.go:                             if condition.Type == "Recommended" {
origin/pr/961:pkg/payload/precondition/clusterversion/recommendedupdate_test.go:                                        Type:    "Recommended",
origin/pr/961:pkg/payload/precondition/clusterversion/recommendedupdate_test.go:                                        Type:    "Recommended",
origin/pr/961:pkg/payload/precondition/clusterversion/recommendedupdate_test.go:                                        Type:    "Recommended",
```

Although I'm personally agnostic about whether you pivot the test code, because I don't have a preference between "DRY, Go compiler ensures we have no typos" vs. "some repetition of the literal string in tests to ensure we didn't typo a single, DRY variable's value" ;)

Reply:
Replaced further strings, but I kept the literals in tests, I have minor preference for explicit values in tests.